src/components/App.js

```diff
@@ -96,12 +98,12 @@ class App extends Component {
     }
 
     // save scrapePropNames
-    this.saveScrapePropNames = () => {
-      let newArr = this.state.scrapePropNames;
-      newArr.push(this.state.segmentPropValue);
-      this.setState({scrapePropNames: newArr});
-      console.log(this.state.scrapePropNames)
-    }
+    // this.saveScrapePropNames = () => {
```

Comment:
Don't need to keep commented code. We can always refer to the old commits if we need to see it again.